3_postnatal_care.do

```diff
@@ -54,9 +55,14 @@ c_pnc_eff2_q
 		}
 		if inlist(country_name,"CostaRica2018") {
 			global pnc "a b i"
-		}		
+		}
+		if inlist(country_name,"Tonga2019") {
```

Comment:
Hmmm, a b c d g seems five categories of skilled professionals for PNC. We might want to double-check.

Reply:
correct, becasue d is community nurse and g is community health worker. i was considering these two cats both in the community health worker part so included.

Reply:
its very strange. the code in my local side changed to only included "a". is there anything related to rebase or something?

Reply:
Let's walk through this in chat. My hunch is we want to stick to the definitions used in table TM8.3 P139.
If it seems messy to disentangle community health worker from community nurse, we could try to see tabulations and compare against TM8.3 (0.3 health officer and 0.9 community nurse)

Reply:
checked;

g - community health worker
removed